### ISSUE TITLE
Only part of nodes have label, how to construct my input? 

### ISSUE THREAD

**Issue by llgaiwyy** [rusty1s/pytorch_geometric#665]
## ❓ Questions & Help
Hi, Sorry to bother you with this question. In my train and validation datasets, only part of nodes have label, how can I construct the input y when I use the model GCN or GAT?

**Comment by rusty1s** [comment]
I suggest holding a mask for each graph which holds information whether a label is present for a node. You can then compute the loss via
```python
out = model(...)
loss = F.nll_loss(out[data.y_mask], data.y)
```

**Comment by llgaiwyy** [comment]
Thanks for your reply, it works, and thanks for your great work!
